src/open-data/Services.csv

```diff
@@ -1,358 +1,358 @@
-playfabbillingservice,playfabbillingservice/messages,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/partyconnectivity,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/partyvoice,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/playstream,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/reads,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/storage,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/telemetry,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/totalexecution,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-playfabbillingservice,playfabbillingservice/writes,Azure PlayFab,Media,Microsoft,Cloud Provider,Cloud,SaaS
-southafricanorth,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-southcentralus,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-southeastasia,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-southindia,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-uaenorth,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-uksouth,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-ukwest,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-virtual network,microsoft.network/connections,Virtual Network,Networking,Microsoft,Cloud Provider,Cloud,IaaS
-virtual network,microsoft.network/p2svpngateways,VPN Gateway,Networking,Microsoft,Cloud Provider,Cloud,IaaS
-virtual network,microsoft.network/virtualnetworkgateways,Virtual Network,Networking,Microsoft,Cloud Provider,Cloud,IaaS
-virtual network,microsoft.network/vpngateways,VPN Gateway,Networking,Microsoft,Cloud Provider,Cloud,IaaS
-virtualmachines,microsoft.security/pricings,Virtual Machines,Compute,Microsoft,Cloud Provider,Cloud,PaaS
-westcentralus,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-westeurope,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-westindia,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-westus,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
-westus2,microsoft.sql/locations,Azure SQL Database,Databases,Microsoft,Cloud Provider,Cloud,PaaS
+ConsumedService,ResourceType,ServiceName,ServiceCategory,ServiceSubcategory,Proposed,PublisherName,PublisherType,Environment,ServiceModel

```

Comment:
Waiting on closure of the FOCUS ServiceSubcategory spec to remove the proposed values.